src/jsx/import/content/register.jsx

```diff
@@ -31,168 +31,169 @@ var RegisterForm = React.createClass({
+        const check = !!checkValidEmail(email);
         if (!check) {
-            const action = setSpinner({'spinner': false});
+            const action = setSpinner({ spinner: false });
             this.props.dispatchSpinner(action);
         }
 
-        this.setState({validated_email_server: true});
-        this.setState({validated_email: check});
-        this.setState({value_email: email});
-    },
-    validatePassword: function(event) {
+        this.setState({ validated_email_server: true });
+        this.setState({ validated_email: check });
+        this.setState({ value_email: email });
+    }
+    validatePassword(event) {
         const password = event.target.value;
-        const check = checkValidPassword(password) ? true : false;
+        const check = !!checkValidPassword(password);
         if (!check) {
-            const action = setSpinner({'spinner': false});
+            const action = setSpinner({ spinner: false });
             this.props.dispatchSpinner(action);
         }
 
-        this.setState({validated_password_server: true});
-        this.setState({validated_password: check});
-        this.setState({value_password: password});
-    },
-  // triggered when 'state properties' change
-    render: function() {
-      // frontend validation
-        var usernameClass = this.state.validated_username ?  '' : 'invalid';
-        var passwordClass = this.state.validated_password ? '' : 'invalid';
+        this.setState({ validated_password_server: true });
+        this.setState({ validated_password: check });
+        this.setState({ value_password: password });
+    }
+    // triggered when 'state properties' change
+    render() {
+        // frontend validation
+        const usernameClass = this.state.validated_username ? '' : 'invalid';
+        const passwordClass = this.state.validated_password ? '' : 'invalid';
 
         if (this.state.validated_email) {
             var emailClass = '';
             var emailNote = '';
         } else {
             var emailClass = 'invalid';
-            var emailNote = <span className={emailClass}>
+            var emailNote = (<span className={emailClass}>
                 Please provide a valid email.
-            </span>;
+                             </span>);
         }
 
-      // backend validation
+        // backend validation
         if (!this.state.validated_password_server) {
-            var passwordNote = <span className='invalid'>
+            var passwordNote = (<span className='invalid'>
                 (Password requirement not met)
-            </span>;
+                                </span>);
         } else {
             var passwordNote = null;
         }
 
         if (!this.state.validated_username_server) {
-            var usernameNote = <span className='invalid'>
+            var usernameNote = (<span className='invalid'>
                 (Username is taken)
-            </span>;
+                                </span>);
```

Comment:
Indentation here seems off.

Reply:
I will try to fix those indentation issues